### PR TITLE
Update ConnectionMethodVariant versions for all plugins

### PR DESCRIPTION
**Pull request by sunithasomanna**
Update ConnectionMethodVariant versions for all plugins in Getting Started Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2997,9 +2997,9 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 
 Example:
 
-        connectionMethodConf:
+        3connectionMethodConf:
```
Comment (jeevan-kamkar) [inline README.md]: it should be connectionMethodConf not 3connectionMethodConf update the same

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested chages

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: fixed

**Review by jeevan-kamkar — APPROVED** [review]

**Comment by sunithasomanna** [comment]
Closing this PR because it will not be taken for the phase 4. Retaining the existing versions for ConnectionMethodVariant parameter for all plugins.

**Pull request closed without merging**